tests/documentation/test_files/DocumentationStyleBear/bad_file.py.test.correct

```diff
@@ -0,0 +1,12 @@
+def improper_indents(indents):
+    """
+    contains improper indents and no following empty line below
```

Comment:
lowercase `c` here makes it a pep 257 violation.
Please run pydocstyle on your test cases to make sure you good files are actually good files. I've asked this many times on this PR.